ui/src/test/resources/user_no_country_response.json

```diff
@@ -0,0 +1,25 @@
+{
+  "token": "usr-f9154016-94e8-4686-a840-075688ac07b5",
+  "status": "PRE_ACTIVATED",
+  "createdOn": "2017-10-30T22:15:45",
```

Comment:
@peter-joseph Could you change 
token to "usr-123", 
created on to "2019-05-23T22:15:45"
program token to "prg-234"
and href to local host?